src/app/containers/Figure/index.jsx

```diff
@@ -10,13 +11,18 @@ import { RequestContext } from '../../contexts/RequestContext';
 
 const LAZYLOAD_OFFSET = 250; // amount of pixels below the viewport to begin loading the image
 
+/* eslint-disable react/no-danger */
```

Comment:
doesn't this disable it throughout the file? we can be more specific and should

Reply:
Done